llvm/lib/ProfileData/Coverage/CoverageMapping.cpp

```diff
@@ -638,6 +638,10 @@ static unsigned getMaxCounterID(const CounterMappingContext &Ctx,
   unsigned MaxCounterID = 0;
   for (const auto &Region : Record.MappingRegions) {
     MaxCounterID = std::max(MaxCounterID, Ctx.getMaxCounterID(Region.Count));
+    if (Region.Kind == CounterMappingRegion::BranchRegion ||
```

Comment:
I feel like it would be nice to add helpers for Region.Kind just to make the code a little easier to read.

E.g.

```
isBranchRegion(Region) { return Region.Kind == CounterMappingRegion::BranchRegion; } 
isMCDCBranchRegion(Region)
```
Maybe in a follow-up commit?

Reply:
I have 5a5838fba37153adb7885c897131dda09227eb2d

Reply:
OK cool